src/Kafka/ManageKafkaPermissions/ManageKafkaPermissions.tsx

```diff
@@ -190,7 +190,11 @@ export const ManageKafkaPermissions: React.FC<ManageKafkaPermissionsProps> = ({
         </Button>,
       ]}
     >
-      <Form>
+      <Form
+        onKeyPress={(event) => {
+          event.key === "Enter" && event.preventDefault();
```

Comment:
But now, the enter key doesn't work on a lot of elements that should be operable with the Enter key. Whenever you prevent the default behavior of the Enter key, it needs to be in the context of a specific item, not the whole page. That creates a lot of accessibility issues. 

Also, whenever you're handling keyboard events, it's really important to test that the rest of the UI is still operable. You would find a lot of the issues currently present if you spent some time testing the keyboard interaction in the UI, It's also important that you review the keyboard interaction of the PF examples for the components you're using and make sure those interactions are maintained when using the components. The PF examples demonstrate proper behavior for keyboard interaction, but a lot of the implementations of PF components in this PR are not consistent with how the examples on PF.org behave.

Reply:
Following discussion on this, I have removed any manual handling of the enter key press and have instead let the PF component handle it out of the box. This seems to work for all the cases now, our only issue being with the focus changing on selecting a value from the resource menu. I have since raised this issue with the PF team and they are looking into this